create-deploy-pr/action.yml

```diff
@@ -29,6 +29,10 @@ inputs:
     description: "Path to config map in repository"
     required: true
     default: "webstore-app/resources/deployment-cm.yaml"
+  forceCommit:
+    description: "Flag to create deploy commit without PR. If true - create commit into deployBranch and don`t create PR"
+    required: true
+    default: "false"
```

Comment:
Isn't default value redundant? We already defined this parameter as required so it should always get a value.

Reply:
Default value placed for backward compatibility. In this case, the **required** parameter looks redundant.